techniques/css/C45.html

```diff
@@ -173,19 +173,19 @@ <h2>Related Techniques</h2>
 			</ol>
 		</section>
 <section id="tests">
-			<h2>Tests</h2>
+			<h2>検証</h2>
 			<section class="test-procedure" id="procedure">
-				<h3>Procedure</h3>
-				<p>For each user interface component that can receive keyboard focus:</p>
+				<h3>手順</h3>
+				<p>キーボードフォーカスを取得できる各ユーザーインターフェースコンポーネントについて:</p>
 				<ol>
-					<li>Set focus to the interface component using the keyboard (generally, navigating to the component using <kbd>Tab</kbd>/<kbd>Shift+Tab</kbd>)</li>
-          <li>Verify that once the component has received focus, a focus indicator is visible</li>
+					<li>キーボードを使用してインタフェースコンポーネントにフォーカスを設定する (一般的には、<kbd>Tab</kbd> 及び <kbd>Shift+Tab</kbd> キーを使用してコンポーネントに移動する)。</li>
+          <li>コンポーネントがフォーカスを取得したら、フォーカスインジケータが可視であることを検証する。</li>
 				</ol>
 			</section>
 			<section class="test-results" id="expected-results">
-				<h3>Expected Results</h3>
+				<h3>期待される結果</h3>
 				<ul>
-					<li>Check #2 is true.</li>
+					<li>チェック 2 が真である。</li>
```

Comment:
ここの表現ですが、既存の他のテクニック (WCAG 2.1 達成方法集) では「#1 の結果が真である。」のような表現になっています。
```suggestion
					<li>#2 の結果が真である。</li>
```